File: php/DecimalToBinary.php
<?php

function binary($decimal) {
    $left;
    $result;

    if ($decimal <= 1) {
        echo (int)$decimal;
        return;
    }

    $left = $decimal % 2;
    $result = $decimal / 2;
    binary($result);
    echo (int)$left;
}

binary(50);
?>

File: net/DecimalToBinary.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace DecimalToBinary
{
    class Program
    {
        static void binary(int number)
        {
            int left, result;
            if (number <= 1)
            {
                Console.Write(number);
                return;
            }
            left = number % 2;
            result = number / 2;
            binary(result);
            Console.Write(left);
        }
        static void Main(string[] args)
        {
            Console.WriteLine("Input number:");
            string dec = Console.ReadLine();
            binary(Convert.ToInt32(dec));
            Console.ReadLine();
        }
    }
}
